plugins/services/src/js/containers/services/__tests__/ServicesTableRenderers-test.js

```diff
@@ -1,181 +1,46 @@
-
-      expect(cpusCell).toEqual(renderNumberCell(1));
-    });
-
-    it("renders sum of resources/stats gpus property", function() {
-      const application = new Application({
-        healthChecks: [{ path: "", protocol: "HTTP" }],
-        cpus: 1,
-        gpus: 1,
-        instances: 2,
-        mem: 2048,
-        disk: 0,
-        tasksStaged: 0,
-        tasksRunning: 2,
-        tasksHealthy: 2,
-        tasksUnhealthy: 0,
-        getResources: jest.fn()
-      });
-
-      application.getResources.mockReturnValue({ gpus: application.gpus });
-      var gpusCell = gpuRenderer(application);
-
-      expect(gpusCell).toEqual(renderNumberCell(application.gpus));
-    });
-
-    it("renders sum of resources/stats mem property", function() {
-      const application = new Application({
-        healthChecks: [{ path: "", protocol: "HTTP" }],
-        cpus: 1,
-        gpus: 1,
-        instances: 2,
-        mem: 2048,
-        disk: 0,
-        tasksStaged: 0,
-        tasksRunning: 2,
-        tasksHealthy: 2,
-        tasksUnhealthy: 0,
-        getResources: jest.fn()
-      });
-
-      application.getResources.mockReturnValue({ mem: application.mem });
-      var memCell = memRenderer(application);
-
-      expect(memCell).toEqual(renderNumberCell("2 GiB"));
-    });
-
-    it("renders sum of resources/stats disk property", function() {
-      const application = new Application({
-        healthChecks: [{ path: "", protocol: "HTTP" }],
-        cpus: 1,
-        gpus: 1,
-        instances: 2,
-        mem: 2048,
-        disk: 0,
-        tasksStaged: 0,
-        tasksRunning: 2,
-        tasksHealthy: 2,
-        tasksUnhealthy: 0,
-        getResources: jest.fn()
-      });
-
-      application.getResources.mockReturnValue({ disk: application.disk });
-      var diskCell = diskRenderer(application);
-
-      expect(diskCell).toEqual(renderNumberCell("0 B"));
-    });
-  });
-
-  describe("#renderRegions", function() {
-    const renderRegions = regionRenderer;
-    let mockService;
-    beforeEach(function() {
-      mockService = {
-        getRegions: jest.fn()
-      };
-    });
-
-    it("renders with no regions", function() {
-      mockService.getRegions.mockReturnValue([]);
-
       expect(
-        renderer.create(renderRegions(mockService)).toJSON()
+        renderer.create(<ServiceMem resource="1" />).toJSON()
       ).toMatchSnapshot();
     });
 
-    it("renders with one region", function() {
-      mockService.getRegions.mockReturnValue(["aws/eu-central-1"]);
-
+    it("renders resources/stats disk property", function() {
       expect(
-        renderer.create(renderRegions(mockService)).toJSON()
+        renderer.create(<ServiceDisk resource="1" />).toJSON()
       ).toMatchSnapshot();
     });
+  });
 
-    it("renders with multiple regions", function() {
-      mockService.getRegions.mockReturnValue(["aws/eu-central-1", "dc-east"]);
-
+  describe("#renderRegions", function() {
```

Comment:
why change to a single region?

Reply:
Is your concern that there is no test for multiple regions?

Reply:
i have no concern here. i just saw that the test checked for multiple regions before and now does not anymore and was curious about why that is.

Reply:
after our offline-discussion, i'd agree that these tests are more likely to create churn than save us from mistakes.